util/package.js

```diff
@@ -1,5 +1,20 @@
 'use strict';
 
+var allStars = require( 'all-stars' );
+var objectAssign = require( 'object-assign' );
+
+/**
+ * Check all-stars db for given person and assign to person if found.
+ *
+ * @param  {Object} person object representing author or maintainer
+ *
+ * @return {Object}        same object given, possibly modified or augmented
+ */
+function getAllStar( person ) {
+  // override properties from all-stars if available
+  return objectAssign( person, allStars( person ) );
+}
```

Comment:
> Rod Vagg <r@va.gg> (npm: rvagg, GitHub: rvagg, Twitter: rvagg)

The representation thing is something related to the given reporter in `credits-cli`.

In a perfect world I'd like to have name, email, npm, github, twitter as strings. 
Projects might not be necessary for now.

> BTW, did you see this comment about assigning? Are you ok returning all that stuff?

Jup - that's why I am asking for the `summary` call in `all-stars`. This will pretty much do the job, without exposing too much data, no?